templates/branding/openqa.suse.de/sponsorbox.html.ep

```diff
@@ -0,0 +1,5 @@
+<div class="text-right" id="sponsorbox">
```

Comment:
It sure does. `templates/branding/openqa.suse.de/sponsorbox.html.ep` differs from openSUSE one by the missing "sponsor" subscript. I can also just reuse the openSUSE one if you prefer